WordPress/src/main/java/org/wordpress/android/util/image/GlidePopTransitionOptions.kt

```diff
@@ -0,0 +1,41 @@
+package org.wordpress.android.util.image
+
+import android.graphics.drawable.Drawable
+import android.view.animation.AnimationUtils
+import com.bumptech.glide.TransitionOptions
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.request.transition.NoTransition
+import com.bumptech.glide.request.transition.Transition
+import com.bumptech.glide.request.transition.TransitionFactory
+import org.wordpress.android.R
+import org.wordpress.android.util.AppLog
+import java.lang.RuntimeException
+
+object GlidePopTransitionOptions : TransitionOptions<GlidePopTransitionOptions, Drawable>() {
```

Comment:
I used [DrawableTransitionOptions](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/resource/drawable/DrawableTransitionOptions.java) as a guide for this implementation.